apps/sms-otp-authentication-portal/src/main/webapp/smsotp.jsp

```diff
@@ -96,7 +96,7 @@
                     <% } %>
                     <div class="error-msg"></div>
                     <div class="segment-form">
-                        <form class="ui large form" id="pin_form" name="pin_form" action="../../commonauth"  method="POST">
+                        <form class="ui large form" id="pin_form" name="pin_form" action="../commonauth"  method="POST">
```

Comment:
Will this work when tenant qualified URL is disabled as well ?

Reply:
yes

Reply:
https://github.com/wso2/identity-apps/blob/master/apps/authentication-portal/src/main/webapp/emailOtp.jsp#L95